core/vm/interpreter.go

```diff
@@ -66,9 +66,20 @@ type Interpreter interface {
 // ScopeContext contains the things that are per-call, such as stack and memory,
 // but not transients like pc and gas
 type ScopeContext struct {
-	Memory   *Memory
-	Stack    *stack.Stack
-	Contract *Contract
+	Memory        *Memory
+	Stack         *stack.Stack
```

Comment:
Go-ethereum is using [Stack](https://github.com/ethereum/go-ethereum/blob/b3f36e10766956bf204f2d2b9415dcd8cfd2be6b/core/vm/stack.go). Probably at some point they moved the code and Erigon hasn't followed.

Reply:
Are you thinking that moving to a homegrown `Stack` is outside the scope of this changeset?

Reply:
Not sure,  but thinking that it might be easier for Erigon team to review only the EOF parts. Later we can make PRs that change stuff like `Stack`.
What do you think?

Reply:
Yep, SGTM. Let's consider it out of scope.